docs/01.GETTING-STARTED.md

```diff
@@ -4,64 +4,70 @@ Currently, only Ubuntu 14.04+ is officially supported as primary development env
 
 There are several dependencies, that should be installed manually. The following list is required for building:
 
-- `gcc` higher than `4.8.2`
+- `gcc` >= `4.8.2`
   - native
   - arm-none-eabi
-- `cmake` higher than `2.8.12.2`
-- `make` higher than `3.81`
-- `bash` higher than `4.3.11`
-- `cppcheck` higher than 1.61
-- `vera++` higher than 1.2.1
+- `cmake` >= `2.8.12.2`
+- `make` >= `3.81`
+- `bash` >= `4.3.11`
+- `cppcheck` >= `1.61`
+- `vera++` >= `1.2.1`
+- `python` >= `2.7.6`
 
 ```bash
-sudo apt-get install gcc g++ gcc-arm-none-eabi cmake cppcheck vera++
+sudo apt-get install gcc g++ gcc-arm-none-eabi cmake cppcheck vera++ python
 ```
 
 To make our scripts run correctly, several shell utilities should be available on the system:
 
 - `find`
-- `bc`
 - `awk`
 - `sed`
-- `sha256sum`
```

Comment:
There are scripts eg.: `run-perf-test.sh` which uses the `awk`. I have not found any script that still uses `sed˙ so I removed it from the doc now.